agent/client.go

```diff
@@ -116,6 +117,23 @@ func (c *CanaryClient) CreateServer(srv *Server) (string, error) {
 	return respServer.UUID, nil
 }
 
+func (client *CanaryClient) FetchUpgradeablePackages() (map[string]string, error) {
+	respBody, err := client.get(ApiServerPath(client.server.UUID))
```

Comment:
Are we going to create a new endpoint for this or parse the existing GET /server response?